routes/api.js

```diff
@@ -231,11 +233,13 @@ router.post('/user-access', function(req, res, next) {
     };
     if(anyOneOnMachine.user === null) {
       stationLog.addUser(res.locals.obj.user, res.locals.obj.badge, req.headers['station-id']);
+      db.logEvent('log on station', + req.headers['stationd-id'] + 'by user' + res.locals.obj.user).then(function(){
```

Comment:
734 log on station NaNby userB1,A1

Like I stated above this log also needs to be edited to include proper spacing between words.  It also looks like the req.headers['stationd-id'] and res.locals.obj.user are not working as you had assumed they would.